GiEnJul/Infrastructure/AutoMapperProfile.cs

```diff
@@ -8,26 +8,25 @@ public AutoMapperProfile()
                 .ForMember(dest => dest.GiverEmail, opt => opt.MapFrom(src => src.Email))
                 .ForMember(dest => dest.GiverFullName, opt => opt.MapFrom(src => src.FullName))
                 .ForMember(dest => dest.GiverLocation, opt => opt.MapFrom(src => src.Location))
                 .ForMember(dest => dest.GiverPhoneNumber, opt => opt.MapFrom(src => src.PhoneNumber))
-                .ForMember(dest => dest.MaxReceivers, opt => opt.MapFrom(src => src.MaxRecievers))
-                .ForMember(dest => dest.SubmitterEmail, opt => opt.MapFrom(src => src.Receiver.ContactEmail))
-                .ForMember(dest => dest.SubmitterFullName, opt => opt.MapFrom(src => src.Receiver.ContactFullName))
-                .ForMember(dest => dest.SubmitterPhoneNumber, opt => opt.MapFrom(src => src.Receiver.ContactPhoneNumber))
-                .ForMember(dest => dest.Institution, opt => opt.MapFrom(src => src.Receiver.Institution))
-                .ForMember(dest => dest.ReferenceId, opt => opt.MapFrom(src => src.Receiver.ReferenceId))
-                .ForMember(dest => dest.ReceiverLocation, opt => opt.MapFrom(src => src.Receiver.Location))
-                .ForMember(dest => dest.Dinner, opt => opt.MapFrom(src => src.Receiver.Dinner))
-                .ForMember(dest => dest.Dessert, opt => opt.MapFrom(src => src.Receiver.Dessert))
-                .ForMember(dest => dest.Note, opt => opt.MapFrom(src => src.Receiver.Note))
-                .ForMember(dest => dest.Event, opt => opt.MapFrom(src => src.Receiver.Event))
-                .ForMember(dest => dest.PersonCount, opt => opt.MapFrom(src => src.Receiver.FamilyMembers.Count));
+                .ForMember(dest => dest.MaxReceivers, opt => opt.MapFrom(src => src.MaxRecievers));
+                //.ForMember(dest => dest.SubmitterEmail, opt => opt.MapFrom(src => src.Receiver.ContactEmail))
+                //.ForMember(dest => dest.SubmitterFullName, opt => opt.MapFrom(src => src.Receiver.ContactFullName))
+                //.ForMember(dest => dest.SubmitterPhoneNumber, opt => opt.MapFrom(src => src.Receiver.ContactPhoneNumber))
+                //.ForMember(dest => dest.Institution, opt => opt.MapFrom(src => src.Receiver.Institution))
+                //.ForMember(dest => dest.ReferenceId, opt => opt.MapFrom(src => src.Receiver.ReferenceId))
+                //.ForMember(dest => dest.ReceiverLocation, opt => opt.MapFrom(src => src.Receiver.Location))
+                //.ForMember(dest => dest.Dinner, opt => opt.MapFrom(src => src.Receiver.Dinner))
+                //.ForMember(dest => dest.Dessert, opt => opt.MapFrom(src => src.Receiver.Dessert))
+                //.ForMember(dest => dest.Note, opt => opt.MapFrom(src => src.Receiver.Note))
+                //.ForMember(dest => dest.Event, opt => opt.MapFrom(src => src.Receiver.Event))
+                //.ForMember(dest => dest.PersonCount, opt => opt.MapFrom(src => src.Receiver.FamilyMembers.Count));
+            
```

Comment:
For future ref: 
It's good to write a short comment explaining why this code is disabled and not removed, possibly with reference to the issue that caused this.